client/app/dedicated/server/task/dedicated-server-task.controller.js

```diff
@@ -1,23 +1,33 @@
 angular.module('App')
   .controller('TaskCtrl', class DedicatedServerTaskController {
-    constructor($scope, $stateParams, Alerter, Server, TASK_STATUS) {
+    constructor($scope, $stateParams, $translate, Alerter, Server) {
       this.$scope = $scope;
       this.$stateParams = $stateParams;
+      this.$translate = $translate;
       this.Alerter = Alerter;
       this.Server = Server;
-      this.TASK_STATUS = TASK_STATUS;
+    }
+
+    $onInit() {
+      this.isServiceExpired = false;
     }
 
     loadDatagridTasks({ offset, pageSize }) {
+      const serviceExpiredErrorCode = 460;
       return this.Server.getTasks(this.$stateParams.productId, pageSize, offset - 1)
         .then(result => ({
           data: _.get(result, 'list.results'),
           meta: {
             totalCount: result.count,
           },
         }))
-        .catch((err) => {
-          this.Alerter.alertFromSWS(this.$translate.instant('server_configuration_task_loading_error'), err, 'taskAlert');
+        .catch((error) => {
+          if (error.code === serviceExpiredErrorCode) {
+            this.isServiceExpired = true;
+            this.Alerter.error(this.$translate.instant('server_configuration_service_expired'), 'taskAlert');
```

Comment:
It's not an error, it's an `info`, don't you think ?

Reply:
You're right, it's not an error, but it should be a warning, because we must get the user attention and tell him his service is expired